.github/workflows/cloud-deploy-embed.yml

```diff
@@ -0,0 +1,51 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Deploy EMBED
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  workflow_call:
+    inputs:
+      env:
+        description: 'Deployment environment - choose between dev and prod'
+        required: true
+        type: string
+jobs:
+  # This workflow contains a single job called "build"
+  deploy_embed:
+    if: ${{ contains(inputs.env, 'dev') }}
```

Comment:
```suggestion
    if: ${{ contains(inputs.env, 'staging') }}
```

Reply:
@SokratisVidros Only one concern about it. Are we going to rename `dev` to `staging` inside of all cloud deployment files? Because we have a lot of code pieces where we use environment GitHub, Netlify. If we will change this we have to add one more variable gh_environment/nf_environment to continue use this environments within following systems. Or we should create for instance Stagging environment within GitHub and fill it in the same secrets and variables as the Development environment already has. Becuse the most part of jobs uses `env: ${{ needs.pre-release.outputs.environment }}` and in the lowest level this variable as a 
```
jobs:
  reusable_embed_deploy:
    runs-on: ubuntu-latest
    timeout-minutes: 80
    environment: ${{ inputs.environment }}
```
or
```
jobs:
  reusable_web_deploy:
    runs-on: ubuntu-latest
    timeout-minutes: 80
    environment: ${{ inputs.environment }}
```
or
```
  reusable_docker:
    needs: [job-setup]
    runs-on: ubuntu-latest
    timeout-minutes: 80
    environment: ${{ inputs.environment }}
```
or
```
  infrastructure_data:
    runs-on: ubuntu-latest
    timeout-minutes: 80
    environment: ${{ inputs.environment }}
```